templates/client-daemonset.yaml

```diff
@@ -28,6 +28,9 @@ spec:
         hasDNS: "true"
       annotations:
         "consul.hashicorp.com/connect-inject": "false"
+        {{- if .Values.client.annotations }}
+{{ toYaml .Values.client.annotations | indent 8 }}
```

Comment:
You can use nindent which allows easier readability see https://github.com/hashicorp/consul-helm/blob/master/templates/server-statefulset.yaml#L43